src/Obj.hs

```diff
@@ -199,6 +199,11 @@ isSym :: XObj -> Bool
 isSym (XObj (Sym (SymPath _ _) _) _ _) = True
 isSym _ = False
 
+isSpecialSym :: XObj -> Bool
```

Comment:
A lot of languages use "reserved", but maybe "special" is better in Lisp? I know it's usually called "special form".

Reply:
I also know it as special, but we can switch to "reserved"; I just feel like that sounds like we’re namesquatting.

Reply:
Right, let's keep it as "special" then.